apps/portals/adknowledgeportal/src/config/explorePageRoutes.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { RouteObject } from 'react-router-dom'
+import { programsSql } from './resources'
+import { computationalToolsQueryWrapperPlotNavProps } from './synapseConfigs/computational_tools'
+import { dataQueryWrapperPlotNavProps } from './synapseConfigs/data'
+import { experimentalToolsQueryWrapperPlotNavProps } from './synapseConfigs/experimental_tools'
+import { peopleQueryWrapperPlotNavProps } from './synapseConfigs/people'
+import programs from './synapseConfigs/programs'
+import { projectsQueryWrapperPlotNavProps } from './synapseConfigs/projects'
+import { publicationsQueryWrapperPlotNavProps } from './synapseConfigs/publications'
+import { results } from './synapseConfigs/results'
+import { studiesQueryWrapperPlotNavProps } from './synapseConfigs/studies'
+import { targetEnablingResourcesQueryWrapperPlotNavProps } from './synapseConfigs/target_enabling_resources'
+import {
+  CardContainerLogic,
+  MarkdownSynapse,
+  QueryWrapperPlotNav,
+} from 'synapse-react-client'
+
+export const explorePageRoutes: RouteObject[] = [
```

Comment:
Replaces routeControlWrapperProps. Defines the routes and route contents for the explore page.